.pre-commit-config.yaml

```
@@ -121,6 +110,43 @@ repos:
         alias: matlab-cpp-format
         files: >-
           ^matlab/src/cpp/
+  - repo: https://github.com/hhatto/autopep8
+    rev: v2.3.2
+    hooks:
+      - id: autopep8
+        name: Python Format
+        alias: python-format
+        args:
+          - "--global-config"
+          - "python/setup.cfg"
+          - "--ignore-local-config"
+          - "--in-place"
```

should we use `--diff` instead of `--in-place`?
>  -d, --diff            print the diff for the fixed source
>  -i, --in-place        make changes to files in place
```suggestion
          - "--diff"
```

---

For a second I just thought on CI but this is run locally, sure, `--in-place` is better for that.

---

Yes.
pre-commit shows diff on format error. So autopep8 doesn't need to show diff by itself.